examples/cloud_run_cloudevents/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use the official Python image.
```

Comment:
nit: In GoogleCloudPlatform we tend to use the term "samples" instead of "examples".

Reply:
May want to move this nit to a different PR just to keep diff counts down. This would require several path changes in README's and whatnot.

Reply:
SGTM, I hadn't realized it was previously in use.